.github/actions/create-upload-suggestions/action.yml

```diff
@@ -0,0 +1,241 @@
+# from https://github.com/OSGeo/grass/blob/main/.github/actions/create-upload-suggestions/action.yml
```

Comment:
Are there any changes in this action?
If not, is it possible to remove it here and directly use the action from GRASS GIS in the linting workflow?

Reply:
No, the files are identical. Using the the action directly from the GRASS repo schould be fine (at least in this version).